README.md

```diff
@@ -1,26 +1,98 @@
 # Loop Exercises
 
 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
-2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num". 
+
+let num = 10
+while (num >= 1) {
+  console.log('the number is: ' + num)
+  num -= 1
+}
+
+
+2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num".
 For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
-3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num". 
+
+num = 10
+for (let i = 0; i <= num; i++ )
+  if (i % 2 === 0 ) {
+  console.log( i + " is even" )
+} else {
+console.log(i + 'is odd') }
+
```

Comment:
I want to see a few more curly brackets here! Take a look [here](https://www.w3schools.com/js/js_conventions.asp) if you want help with formatting/styling.